documentation/ipi-install/modules/ipi-install-additional-install-config-parameters.adoc

```diff
@@ -5,7 +5,7 @@
 [id="additional-install-config-parameters_{context}"]
 = Additional `install-config` parameters
 
-This topic describes the required parameters, the `hosts` parameter, 
+This topic describes the required parameters, the `hosts` parameter,
 and the `bmc` `address` parameter for the `install-config.yaml` file.
```

Comment:
This section describes the required parameters and optional parameters that are part of the `install-config.yaml` file